storage/testqueue/test.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package testqueue
```

Comment:
Let's put the tests inside `testsuite` that way when using `testqueue` it won't bring in testing packages.